src/api/ApiResponder.php

```diff
@@ -22,10 +22,13 @@ class ApiResponder {
     protected $isTest;
 
     /**
-     * functions which allow access by unauthenticated users
-     * For now, all game functionality should require login: only
-     * add things to this list if they are necessary for user
-     * creation and/or login.
+     * Functions which allow access by unauthenticated users.
+     *
+     * All game functionality should require login.
+     *
+     * Only add things to this list if they are necessary for user
+     * creation and/or login, or for any content not requiring
+     * login, like help.
```

Comment:
You say somewhere deeper in the code that help functionality should be designed not to make database calls.  Can you say that in this comment too, as a reminder to Future Us to limit DB-using functionality accessible to non-logged-in users?